client/wfnews-war/src/main/angular/src/app/components/full-details/area-restrictions-full-details/area-restrictions-full-details.component.ts

```diff
@@ -88,26 +89,33 @@ export class AreaRestrictionsFullDetailsComponent implements OnInit {
       if (this.incident.fireOfNoteInd) {
         L.marker(location, { icon: fireOfNoteIcon }).addTo(this.map);
       } else {
-        let colorToDisplay;
-        switch (this.incident.stageOfControlCode) {
-          case 'OUT_CNTRL':
-            colorToDisplay = '#FF0000'
-            break;
-          case 'HOLDING':
-            colorToDisplay = '#ffff00'
-            break;
-          case 'UNDR_CNTRL':
-            colorToDisplay = '#98E600'
-            break;
-          case 'OUT':
-            colorToDisplay = '#999999'
-            break;
-          default:
-            colorToDisplay = 'white';
-        }
+        const colorToDisplay = this.setDisplayColor(this.incident.stageOfControlCode)
         L.circleMarker(location, { radius: 15, fillOpacity: 1, color: 'black', fillColor: colorToDisplay }).addTo(this.map)
       }
     }
+
+    // now fetch the rest of the incidents in the area and display on map
+    try {
+        const locationData = new LocationData()
+        locationData.latitude = Number(this.restrictionData.centroidLatitude);
+        locationData.longitude = Number(this.restrictionData.centroidLongitude);
+        locationData.radius = 10;
+        const stageOfControlCodes = ['OUT_CNTRL', 'HOLDING', 'UNDR_CNTRL'];
+        const incidents = await this.publishedIncidentService.fetchPublishedIncidentsList(0, 9999, locationData, null, null, stageOfControlCodes).toPromise()
+        if (incidents?.collection && incidents?.collection?.length > 0){
+          for (const item of incidents.collection) {
+            const location = [Number(item.latitude), Number(item.longitude)]
+            if (item.fireOfNoteInd) {
+              L.marker(location, { icon: fireOfNoteIcon }).addTo(this.map);
+            } else {
+              const colorToDisplay = this.setDisplayColor(item.stageOfControlCode)
+              L.circleMarker(location, { radius: 5, fillOpacity: 1, color: 'black', fillColor: colorToDisplay }).addTo(this.map)
+            }
+          }
+      }
+    } catch(err){
+      console.log('Could not retrieve surroundung incidents for area restriction')
```

Comment:
If we are not doing anything when this happens, would suggest using console.error() to make it more obvious in log.

Reply:
Done, thanks